server/src/db/seeders/AttractionSeeder.js

```diff
@@ -5,24 +5,24 @@ class AttractionSeeder {
     const attractionsData = [
       {
         name: "Baha'i Gardens",
-        description: "A very beautiful garden by the ocean."
+        description: "A very beautiful garden by the ocean.",
       },
       {
         name: "Jean's House",
-        description: "Home to million dollar apps and the world's greatest Overwatch player."
+        description: "Home to million dollar apps and the world's greatest Overwatch player.",
       },
       {
         name: "Launch Academy",
-        description: "Jean-Free Zone"
-      }
-    ]
-    for(const singleAttractionData of attractionsData) {
+        description: "Jean-Free Zone",
+      },
```

Comment:
I love the goofiness, but I would seed data that is slightly less jokey since employers will get confused